app/controllers/admins/employees_controller.rb

```diff
@@ -20,27 +24,26 @@ def update
     end
 
     def destroy
-        @kudos = Kudo.where(['giver_id = ? or receiver_id = ?', @employee.id, @employee.id]).all
-        @kudos.each(&:destroy)
-        @employee.destroy
-        redirect_to admins_employees_url, notice: 'Employee was successfully destroyed.'
+      @kudos = Kudo.where(['giver_id = ? or receiver_id = ?', @employee.id, @employee.id]).all
+      @kudos.each(&:destroy)
+      @employee.destroy
+      redirect_to admins_employees_url, notice: 'Employee was successfully destroyed.'
     end
 
     private
+
     # Use callbacks to share common setup or constraints between actions.
     def set_employee
```

Comment:
similar, to what we have in the company_values
```suggestion
    def employee
```